runtime/compilers/rillv1/parse_model.go

```diff
@@ -16,16 +16,17 @@ import (
 
 // ModelYAML is the raw structure of a Model resource defined in YAML (does not include common fields)
 type ModelYAML struct {
-	commonYAML        `yaml:",inline" mapstructure:",squash"` // Only to avoid loading common fields into InputProperties
-	Refresh           *ScheduleYAML                           `yaml:"refresh"`
-	Timeout           string                                  `yaml:"timeout"`
-	Incremental       bool                                    `yaml:"incremental"`
-	State             *DataYAML                               `yaml:"state"`
-	Splits            *DataYAML                               `yaml:"splits"`
-	SplitsWatermark   string                                  `yaml:"splits_watermark"`
-	SplitsConcurrency uint                                    `yaml:"splits_concurrency"`
-	InputProperties   map[string]any                          `yaml:",inline" mapstructure:",remain"`
-	Stage             struct {
+	commonYAML            `yaml:",inline" mapstructure:",squash"` // Only to avoid loading common fields into InputProperties
+	Refresh               *ScheduleYAML                           `yaml:"refresh"`
+	Timeout               string                                  `yaml:"timeout"`
+	Incremental           bool                                    `yaml:"incremental"`
+	State                 *DataYAML                               `yaml:"state"`
+	Partitions            *DataYAML                               `yaml:"partitions"`
+	Splits                *DataYAML                               `yaml:"splits"` // Deprecated: use "partitions" instead
+	PartitionsWatermark   string                                  `yaml:"partitions_watermark"`
```

Comment:
missing `splits_watermark` ?

Reply:
No this is intentional – `splits` is the only field anyone uses right now (also it's a beta feature, so breaking changes are okay)